torch/csrc/jit/codegen/cuda/python_frontend/python_bindings.cpp

```diff
@@ -1259,6 +1269,48 @@ void initNvFuserPythonBindings(PyObject* module) {
       py::arg("correction"),
       py::arg("keepdim") = false,
       py::return_value_policy::reference);
+  nvf_ops.def(
+      "batch_norm",
+      [](nvfuser::FusionDefinition::Operators& self,
+         nvfuser::Tensor x,
+         nvfuser::Tensor weight,
+         nvfuser::Tensor bias,
+         nvfuser::Tensor running_mean,
+         nvfuser::Tensor running_var,
+         bool kTraining,
```

Comment:
Nit: Why is the training parameter preficed with a `k`? I thought the `k` usually denoted an enum value.

Reply:
Probably I copied it from https://github.com/pytorch/pytorch/blob/614d6f19e3d30cac0d77059e738d1f25d75eb408/torch/csrc/jit/codegen/cuda/ops/normalization.cpp#L465

I'll rename it.